### DISCUSSION TITLE
Welcome to Full-Stack-Web-developer Discussions!

### DISCUSSION THREAD

**Discussion by AdarshTheki** [AdarshTheki/Full-Stack-Web-developer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by saurabh-s19** [comment]
hello,
i want to contribute in web projects ,but i can't be able to do so please share some knowledge that how can i first.
